build/flaws/bad-bcd-queries.js

```diff
@@ -0,0 +1,27 @@
+const { packageBCD } = require("../resolve-bcd");
+
+// Bad BCD queries are when the `<div class="bc-data">` tags have an
+// ID (or even lack the `id` attribute) that don't match anything in the
+// @mdn/browser-compat-data package. E.g. Something like this:
+//
+//    <div class="bc-data" id="bcd:never.ever.heard.of">
+//
+const getBadBCDQueriesFlaws = (doc, $) =>
```

Comment:
I sometimes much prefer the shorthand-arrow functions, especially one-lines. Here it didn't even affect the indentation. So I happily reverted my unnecessary lambda-ifiying.

Wrt to adding a log, did you know about that VSCode super-power?
![](https://user-images.githubusercontent.com/4051932/115375567-36d20300-a1ce-11eb-927d-c0af3c876af2.mov)

A language based way for logging in any context that I really like (and stole from Rust) is adding something like this to the codebase:
```typescript
declare global {
  const dbg: <V>(value: V) => V;
}

(global as any).dbg = (value: any) => {
  console.log(value);
  return value;
};
```

Then you can just write:
```typescript
[23].filter(n => dbg(n * 2));
```

[Rust's dbg](https://doc.rust-lang.org/std/macro.dbg.html) is even cooler, it would log something like:
```typescript
[main.ts:L3] 23 * 2 = 46
```